src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
I'm pausing here (iteration 1).
Please review usage of suppressions versus assertions. My understanding is that suppressions are reserved for cases where you could figure out something is not null by reading the method (but the compiler analysis is not that capable).